include/deal.II/fe/fe_values.h

```diff
@@ -3774,6 +3784,14 @@ class FEFaceValues : public FEFaceValuesBase<dim, spacedim>
   reinit(const typename Triangulation<dim, spacedim>::cell_iterator &cell,
          const unsigned int                                          face_no);
 
+  /*
+   * Alternative reinitialization function that also only provides information
+   * that does not depend on the global DoF numbering.
```

Comment:
I don't think this sentence is understandable if you don't already know what this function does. Instead of writing it in the negative (what it doesn't provide), I'd say that this function reinitializes the current object using a reference to a cell and face; "given that the cell iterator only points into a triangulation, the same restrictions apply as for the previous function".